back-end/config/loader/AbstractLoader.js

```diff
@@ -0,0 +1,39 @@
+class AbstractLoader {
```

Comment:
`AbstractConfigLoader` makes more sense then here right?

Reply:
Yep :)

Reply:
Hmm come to think of it, it would if status/persister/AbstractPersister would also be called status/persister/AbstractStatusPersister. To keep it consistent, I can either update 0 or 2 abstract classes, but not 1.

Reply:
Disagree. `AbstractConfigLoader` seems spot on here anyway. Same goes for `status/persister/AbstractPersister`.

Reply:
Let's agree to disagree then. Renamed the loader to configLoader whilst leaving the persister as-is.